example/lib/pages/widgets.dart

```diff
@@ -0,0 +1,130 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_map/plugin_api.dart';
+import 'package:latlong/latlong.dart';
+
+import '../pages/zoombuttons_plugin_option.dart';
+import '../widgets/drawer.dart';
+
+class WidgetsPage extends StatelessWidget {
+  static const String route = 'widgets';
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: Text('Widgets')),
+      drawer: buildDrawer(context, WidgetsPage.route),
+      body: Padding(
+        padding: EdgeInsets.all(8.0),
+        child: Column(
+          children: [
+            Flexible(
+              child: FlutterMap(
+                options: MapOptions(
+                  center: LatLng(51.5, -0.09),
+                  zoom: 5.0,
+                  plugins: [
+                    ZoomButtonsPlugin(),
+                  ],
+                ),
+                layers: [
+                  ZoomButtonsPluginOption(
+                    minZoom: 4,
+                    maxZoom: 19,
+                    mini: true,
+                    padding: 10,
+                    alignment: Alignment.bottomLeft,
+                  )
+                ],
+                children: <Widget>[
+                  TileLayerWidget(
+                    options: TileLayerOptions(
+                      urlTemplate:
+                          'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
+                      subdomains: ['a', 'b', 'c'],
+                    ),
+                  ),
+                  MovingWithoutRefreshAllMapMarkers(),
+                  Text(
+                    'Plugin is just Text widget',
+                    style: TextStyle(
+                        fontSize: 22,
+                        color: Colors.red,
+                        fontWeight: FontWeight.bold,
+                        backgroundColor: Colors.yellow),
+                  )
+                ],
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class MovingWithoutRefreshAllMapMarkers extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() =>
+      _MovingWithoutRefreshAllMapMarkersState();
+}
+
+class _MovingWithoutRefreshAllMapMarkersState
+    extends State<MovingWithoutRefreshAllMapMarkers> {
+  Marker _marker;
+  Timer _timer;
+  int _markerIndex = 0;
+
+  @override
+  void initState() {
+    super.initState();
+    _marker = _markers[_markerIndex];
+    _timer = Timer.periodic(Duration(seconds: 1), (_) {
+      setState(() {
+        _marker = _markers[_markerIndex];
+        _markerIndex = (_markerIndex + 1) % _markers.length;
+      });
+    });
+  }
+
+  @override
+  void dispose() {
+    super.dispose();
+    _timer.cancel();
```

Comment:
Could this lead into an issue: canceling the timer after disposing?

Reply:
Fixed :) thx @S-Man42